services/MerchStoreService.ts

```diff
@@ -359,19 +348,40 @@ export default class MerchStoreService {
         };
       }),
       totalCost: order.totalCost,
+      pickupEvent: {
+        ...order.pickupEvent,
+        start: MerchStoreService.dateToHumanReadableDateString(order.pickupEvent.start),
+        end: MerchStoreService.dateToHumanReadableDateString(order.pickupEvent.end),
+      },
     };
     this.emailService.sendOrderConfirmation(user.email, user.firstName, orderConfirmation);
 
     return order.getPublicOrder();
   }
 
-  public async verifyOrder(originalOrder: MerchItemOptionAndQuantity[], user: UserModel): Promise<void> {
+  private static dateToHumanReadableDateString(date: Date): string {
+    return moment(date).format('MMMM d, H:mm A');
+  }
+
+  public async validateOrder(originalOrder: MerchItemOptionAndQuantity[], user: UserModel): Promise<void> {
     return this.transactions.readWrite(async (txn) => {
-      this.verifyOrderWithTransaction(originalOrder, user, txn);
+      this.validateOrderUnderTxn(originalOrder, user, txn);
     });
   }
 
-  private async verifyOrderWithTransaction(originalOrder: MerchItemOptionAndQuantity[],
+  /**
+   * Validates a merch order. An order is considered valid if all the below are true:
+   *  - all the ordered item options exist within the database
+   *  - the ordered item options were placed for non-hidden items
+   *  - the user wouldn't reach monthly or lifetime limits for any item if this order is placed
+   *  - the requested item options are in stock
+   *  - the user has enough credits to place the order
+   * @param originalOrder
+   * @param pickupEventUuid
+   * @param user
+   * @param txn
```

Comment:
These annotations aren't adding anything useful.
```suggestion
   * @param originalOrder
   * @param pickupEventUuid
   * @param user
   * @param txn
```